conf/establishers.routes

```diff
@@ -90,3 +90,5 @@ GET        /:index/individual/previousAddress                       controllers.
 POST       /:index/individual/previousAddress                       controllers.register.establishers.individual.PreviousAddressController.onSubmit(mode: Mode = NormalMode,index:Index)
 GET        /:index/individual/changePreviousAddress                       controllers.register.establishers.individual.PreviousAddressController.onPageLoad(mode: Mode = CheckMode,index:Index)
 POST       /:index/individual/changePreviousAddress                       controllers.register.establishers.individual.PreviousAddressController.onSubmit(mode: Mode = CheckMode,index:Index)
+
+GET        /:index/individual/check-your-answers-individual                          controllers.register.establishers.individual.CheckYourAnswersIndividualController.onPageLoad(index: Index)
```

Comment:
Should this just be `/:index/individual/check-your-answers` as it already indicates that it's about individuals